esp32s2-hal/ld/memory.x

```diff
@@ -11,6 +11,8 @@ ENTRY(ESP32Reset)
 /* reserved at the start of DRAM */
 RESERVE_DRAM = 0x4000;
```

Comment:
Does `RESERVE_DRAM` offset have other purposes than skipping the area reserved for CPU caches?
I'd suggest to rename it to a more intuitive name, e.g. `RESERVE_CACHES`, even because it is not exclusive to DRAM access, but IRAM as well.

If `esp-hal` does not yet configure the CPU caches on bringup, maybe we could change this value to just **0x2000**, which is the default configuration (8kB I-Cache and 0kB D-Cache).

Reply:
that's a good idea